Makefile

```diff
@@ -609,11 +609,11 @@ gcp-apply-binauthz-policy: build/policies/binauthz.yaml
 	gcloud beta $(GCP_PROJECT_FLAG) container binauthz policy import build/policies/binauthz.yaml
 
 all-protos: golang-protos http-proxy-golang-protos swagger-json-docs
-golang-protos: internal/pb/backend.pb.go internal/pb/frontend.pb.go internal/pb/matchfunction.pb.go internal/pb/messages.pb.go internal/pb/mmlogic.pb.go internal/pb/evaluator.pb.go
```

Comment:
`make all-protos` will fail since `api/synchronizer.proto` does not exist

Reply:
Looks like the -a did not add all the new / renamed files. Thanks for catching - uploaded now.